specifyweb/frontend/js_src/lib/localization/attachments.ts

```diff
@@ -143,7 +143,7 @@ export const attachmentsText = createDictionary({
     'en-us': 'Attachment Import Datasets ({count:number})',
   },
   newAttachmentDataset: {
-    'en-us': 'Attachment Import Dataset: New',
+    'en-us': 'New Attachment Dataset {date}',
```

Comment:
forgot the type:

```suggestion
    'en-us': 'New Attachment Dataset {date:string}',
```

(optional) modify localization tests to catch this case (or at least leave a TODO for that)